src/main/java/org/assertj/assertions/generator/BaseAssertionGenerator.java

```diff
@@ -123,8 +123,8 @@ public File generateCustomAssertionFor(ClassDescription classDescription) throws
     buildTargetDirectory(targetDirectory);
     File[] assertionClassesFile = new File[2];
     final String concreteAssertClassFileName = assertClassNameOf(classDescription) + ".java";
-    final String abstractAssertClassFileName = abstractAssertClassNameOf(classDescription) + ".java";
-    assertionClassesFile[0] = createFile(assertionFileContent[0], abstractAssertClassFileName, targetDirectory);
+    final String AbstractObjectAssertClassFileName = AbstractObjectAssertClassNameOf(classDescription) + ".java";
```

Comment:
I prefer to leave it to `abstractAssertClassFileName` because it's named like that as opposed to `concreteAssertClassFileName`